brands/shop/ticket.json

```
@@ -14,6 +14,8 @@
     "count": 52,
```

I would also recommend to add the following values:

```suggestion
     "count": 52,
     "countryCodes": ["de"],
     "match": ["shop/ticket|Reisezentrum"],
```